### PR TITLE
[Notifier] Create `DesktopChannel` and `JoliNotif` bridge

### PR DESCRIPTION
**Pull request by ahmedghanem00**
Updating docs according to symfony/symfony#57683

Fixes https://github.com/symfony/symfony-docs/issues/20168

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
This looks correct to me ... but I'm not an expert here, so I'll wait for more approvals before merging. Thanks!

**Review by OskarStark — APPROVED** [review]
Looks good to me, but a versioned directive is missing for 7.2

**Comment by ahmedghanem00** [comment]
> Looks good to me, but a versioned directive is missing for 7.2

Can you provide an example for this directive? :)

File: notifier.rst
Comment (OskarStark) [inline notifier.rst:695]:
```suggestion

.. versionadded:: 7.2

    The `Desktop` channel was introduced in Symfony 7.2.

```

**Comment by OskarStark** [comment]
> Can you provide an example for this directive? :)

Done sorry :-)

File: notifier.rst
Comment (OskarStark) [inline notifier.rst:641]:
```suggestion

.. versionadded: 7.2

    The JoliNotif bridge was introduced in Symfony 7.2.

```

File: notifier.rst
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The ``Desktop`` channel was introduced in Symfony 7.2.
```

**Review by OskarStark — APPROVED** [review]
Good to merge after my two comments

**Comment by javiereguiluz** [comment]
Merged! Ahmed, thanks a lot for this contribution.

**Pull request merged**
